core/sql/lib_mgmt/src/main/resources/init_libmgmt.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+SERVER_JAR=${MY_SQROOT}/export/lib/lib_mgmt.jar
+CI=sqlci
+CATALOG_NAME=TRAFODION
+LIB_SCHEMA="DB__LIBMGR"
+DB__LIBMGRROLE=DB_LIBMGRROLE
```

Comment:
Did you want it the other way round, one underscore on the left and two underscores on the right? Also change the code below where we reference ${DB__LIBMGRROLE}?